core/templates/domain/collection/collection.model.ts

```diff
@@ -32,35 +32,39 @@ interface ExplorationIdToNodeIndexMap {
 }
 
 export interface CollectionBackendDict {
-  'id': string;
-  'title': string;
-  'objective': string;
-  'language_code': string;
-  'tags': string[];
-  'schema_version': number;
+  // Creating an Empty Collection initializes properties below with
```

Comment:
same comment, I think this is being addressed in #13044

Reply:
Probably a dependency on other files covered here, I did not add it in tsconfig-strict or my PR overview here.